src/app/presentation/components/workspace-components/workspace-menu/workspace-menu.component.html

```diff
@@ -9,16 +9,16 @@
     <ul>
       <li>
         <ul class="group-menu">
-          <li class="menu-item" *ngIf="userWorkspaceInfo.role == 1 || userWorkspaceInfo.role == 2" >
+          <li *appIfUserRoles='[1,2]' class="menu-item">
```

Comment:
Create tech dept for adding permissions on server side and then integrate it here. Use hard code is a bad solution.